native/src/modules/feedback/Feedback.js

```diff
@@ -57,28 +57,26 @@ export type PropsType = {|
   sendingStatus: SendingStatusType,
   onCommentChanged: (comment: string) => void,
   onFeedbackContactMailChanged: (contactMail: string) => void,
-  feedbackOrigin: FeedbackOriginType,
+  isSearchFeedback: boolean,
+  isPositiveFeedback: boolean,
```

Comment:
So we have no less information now? The info if there was nothing found or not enough found got lost, right? 

I think we should keep this functionality, because i think for the future it would be nice to get as much information as possible, so cities can optimize their content and Integreat has more impact.